pathfinder/tests/fixtures/network_service.py

```diff
@@ -56,6 +56,38 @@ def channel_descriptions() -> List:
     return channel_descriptions
 
 
+@pytest.fixture
+def channel_descriptions_case_2() -> List:
+
+    # Now initialize some channels in this network.
+    # The tuples in channel_descriptions define the following:
+    # (
+    #     p1_index,
+    #     p1_deposit,
+    #     p1_transferred_amount,
+    #     p1_fee,
+    #     p2_index,
+    #     p2_deposit,
+    #     p2_transferred_amount,
+    #     p2_fee
+    # )
+    # Topology:
+    #  /----- 1 ----\
+    # 0 -- 2 -- 3 -- 4
+    #       \-- 5 --/
+
+    channel_descriptions = [
+        (0, 100,  20, 0.3, 1,  50,  10, 0.3),  # capacities  90 --  60
+        (1,  40,  10, 0.2, 4, 130, 100, 0.2),  # capacities 130 --  40
+        (0,  90,  10, 0.1, 2,  10,   0, 0.1),  # capacities  80 --  10
+        (2,  50,  20, 0.2, 3,  50,  20, 0.2),  # capacities  50 --  50
+        (3, 100,  40, 0.1, 4,  80,   0, 0.1),  # capacities  60 -- 120
+        (2,  30,  10, 0.1, 5,  40,  15, 0.1),  # capacities  35 --  35
+        (5, 500, 900, 0.1, 4, 750, 950, 0.1),  # capacities 550 -- 700
+    ]
```

Comment:
maybe specify the significant differences between the topologies as a comment, including expected behaviour and a 'tag' what is edgy about the test case.

Reply:
Done.